docs/docs/governance/cross-chain-governance.md

```diff
@@ -0,0 +1,134 @@
+# Cross Chain Governance
+
+To simplify the maintenance and management of the protocol contracts, we use a cross-chain governance that allows for a DAO proposal on mainnet to reach contracts on other networks.
+
+# How it works
+
+To reach other chains, calls emitted from the mainnet DAO go though the [Connext bridge](https://www.connext.network/) and are executed on the other side of the bridge, after a period of cooldown.
+
+```
+(mainnet)                       (destination chain)    (cooldown period)
+DAO proposal >  Connext Bridge >   Safe multisig    >  wait for 7 days   >
+```
+
+The workflow is as follow
+
+1. a DAO proposal is created, containing 1 call per chain
+2. if the vote succeed, the DAO proposal is executed. All calls are send to the bridge
+3. each call cross the bridge seperately towards its destination on a specific chain
+4. the call is received on the destination chain by Unlock multisig
+5. once received, the call is held in the multisig for a period of X days during which it can be cancelled.
+6. once the cooldown period ends, the call is executed
+
+NB: The cooldown period is useful to prevent malicious or errored calls from being executed
+
+The Safes still own Unlock contracts and Proxy Admins on destination networks (no changes in governance here).
+
+# Supported Networks
+
+(as Oct 10, 2023)
+
+|                    | Unlock Protocol | Safe | Connext |
+| ------------------ | --------------- | ---- | ------- |
+| Ethereum Mainnet   | x               | x    | x       |
+| Gnosis             | x               | x    | x       |
+| Polygon            | x               | x    | x       |
+| Optimism           | x               | x    | x       |
+| Arbitrum One       | x               | x    | x       |
+| BNB Chain          | x               | x    | x       |
+| Avalanche          | x               | x    |         |
+| Base               | x               | x    |         |
+| Celo               | x               | x    |         |
+| Palm               | x               |      |         |
+| Polygon zkEVM      |                 | x    |         |
+| zkSync Era mainnet |                 | x    |         |
+| Aurora             |                 | x    |         |
+
+# How to setup the multisig
+
+On every receiving chain, we need a Gnosis Safe configured with two Zodiac plugins
+
+- one Connext bridge receiver
+- one Delay
+
+1. In Gnosis web UI, go to Apps > Zodiac
+2. add the Delay module (cooldown : 7 days, expiration: 90 days)
+3. add the Connext module (Origin sender address : DAO address, Origin domain ID: 6648936 - see [list of Connext domains](https://docs.connext.network/resources/deployments#ethereum))
+4. Go to Connext module > write > `setTarget` with the Delay address
+5. Get the Connext module address then remove the Connext module
+6. Go to Delay > write > `enableModule` with the Connext module address
+
+As code it looks like
+
+```jsx
+safe.disableModule(connextMod.address)
```

Comment:
Here we disable the Connext mod first to make sure it is inactive, because we need to re-enable it by calling `enableModule` from the delay mod directly (vs from the safe) so it becomes linked to the delay and can pass the call from the bridge to the delay mod. 

Not very clear... Maybe clearer with images

On the UI, after being installed it looks like below

![image](https://github.com/unlock-protocol/unlock/assets/743246/e6098dc3-563d-4328-8e44-a32da70c690e)

Then we disable it and re-enable from the dealy module so it appears like this

![image](https://github.com/unlock-protocol/unlock/assets/743246/2688b665-c55a-42c6-b5d4-1c2f8b902b73)